sdk/forms/src/main/java/com/klaviyo/forms/KlaviyoWebViewDelegate.kt

```diff
@@ -15,129 +14,243 @@ import androidx.core.content.ContextCompat.startActivity
 import androidx.webkit.JavaScriptReplyProxy
 import androidx.webkit.WebMessageCompat
 import androidx.webkit.WebViewCompat
+import androidx.webkit.WebViewFeature.DOCUMENT_START_SCRIPT
 import androidx.webkit.WebViewFeature.WEB_MESSAGE_LISTENER
 import androidx.webkit.WebViewFeature.isFeatureSupported
-import com.klaviyo.analytics.DeviceProperties
 import com.klaviyo.analytics.Klaviyo
 import com.klaviyo.analytics.networking.ApiClient
-import com.klaviyo.core.BuildConfig
 import com.klaviyo.core.Registry
 import com.klaviyo.core.config.Clock
 import com.klaviyo.core.utils.WeakReferenceDelegate
+import com.klaviyo.forms.BridgeMessage.Companion.handShakeData
+import java.io.BufferedReader
 
 internal class KlaviyoWebViewDelegate : WebViewClient(), WebViewCompat.WebMessageListener {
     companion object {
+        internal const val BRIDGE_NAME = "KlaviyoNativeBridge"
+
         private const val MIME_TYPE = "text/html"
+
+        private const val LOAD_FROM_FILE = true
```

Comment:
TODO - remove this -- turns out loading from a file URL wasn't the font / CORS issue at all 🤬